src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
 @import "blocks/page";
 @import "blocks/header";
 @import "blocks/catalog";
+@import "blocks/variables";
+
+* {
+  transition: transform ease $transition-duration;
```

Comment:
transition should be placed in .header block, not inside *, pls fix

Reply:
@lerastarynets if I put to .header block it should not affect blocks in .catalog block?